js/src/textures/TextTexture.js

```diff
@@ -0,0 +1,52 @@
+var _ = require('underscore');
+var Promise = require('bluebird');
+var TextTextureBase = require('./TextTexture.autogen');
+
+var TextTextureView = TextTextureBase.TextTextureView.extend({});
+
+var TextTextureModel = TextTextureBase.TextTextureModel.extend({
```

Comment:
This was previously defined, but did not work together with the new autogenned classes, so this is simply a fix of existing functionality. I find it quite useful.